client/root.js

```diff
@@ -85,8 +96,10 @@ async function getLoggedInLandingPage( { dispatch, getState } ) {
 	// ensure that the primary site info is loaded into Redux before proceeding.
 	const primarySiteId = getPrimarySiteId( getState() );
 	await dispatch( waitForSite( primarySiteId ) );
-
 	const primarySiteSlug = getSiteSlug( getState(), primarySiteId );
+	const primarySiteURL = getSiteUrl( getState(), primarySiteId );
+	const adminInterface = getSiteOption( getState(), primarySiteId, 'wpcom_admin_interface' );
```

Comment:
We can use the existing `isAdminInterfaceWPAdmin()` selector here.

Reply:
Thanks